Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = props => (
+    <div className='card-content'>
+        <h2>{props.content.title}</h2>
```

Comment:
great use of props here.. you got it down.

Reply:
I would however just say that you only need props.content. You don't need to treat it the way you treat an actual object by going that deep. The value of props is just a name.. so props.title will just look like title={object.content.tile} on the other end.